tests/lib/metrics_test.py

```diff
@@ -0,0 +1,130 @@
+import time
+from unittest.mock import MagicMock, call, patch
+
+import boto3
+from cloudformation_cli_python_lib.interface import MetricTypes, StandardUnit
+from cloudformation_cli_python_lib.metrics import MetricPublisher, format_dimensions
+
+from botocore.stub import Stubber
+
+# TODO convert patch to fixture
+
+
+class MockSession:
+    def __init__(self, client):
+        self._client = client
+
+    def client(self, name):  # pylint: disable-msg=unused-argument
```

Comment:
you can use `_name` here to mark arguments as unused. pylint will respect that convention

Reply:
Good call, updated.